commons/c2cgeoportal_commons/models/__init__.py

```diff
@@ -34,6 +34,12 @@
 import sqlalchemy.ext.declarative
 import sqlalchemy.ext.declarative.api
 
+try:
+    from pyramid.i18n import TranslationStringFactory
+    _ = TranslationStringFactory('c2cgeoportal_admin')
+except ImportError as e:
+    _ = lambda s: s
```

Comment:
@sbrunner: I blocked here, lingua message extractor does not like python3 typing, mypi does not like python3 wihout typing and flake does not like lambda ...

Reply:
```
def null_translation(s: str) -> str:
        return s
    _ = null_translation
```
should pass the message extractor ...